recipes/setup_server.rb

```diff
@@ -28,4 +28,17 @@
     notifies :restart, "service[apache2]"
 end
 
+template "/etc/init.d/websockify" do
+    source "websockify.erb"
+    owner "root"
+    group "root"
+    variables({
+        :websockify_port => node['abiquo']['websockify']['port'],
+        :websockify_key => node['abiquo']['websockify']['key'],
+        :websockify_crt => node['abiquo']['websockify']['crt']
+    })
+    action :create
+    notifies :restart, "service[websockify]"
+end
```

Comment:
Add a test that makes sure the template is rendered and that notifies the service restart in the [setup_server_spec.rb test](https://github.com/abiquo/abiquo-cookbook/blob/master/spec/setup_server_spec.rb).